client/app/cdn/dedicated/quota/order/cdn-dedicated-quota-order.html

```diff
@@ -1,101 +1,97 @@
-<div data-ng-controller="OrderQuotaCtrl">
+<div data-wizard
```

Comment:
Only previous inline `data-ng-controller` has been changed in this file (+ adding $uibModalInstance.dismiss)